demos/index.html

```diff
@@ -88,7 +89,11 @@
                             gradeNames: ["gpii.firstDiscovery.auxSchema"]
                         },
                         prefsEditor: {
-                            prefsEditorType: "gpii.firstDiscovery.firstDiscoveryEditor"
+                            store: {
```

Comment:
"gpii.firstDiscovery.firstDiscoveryEditor" is an instance of prefsEditorLoader. In the assembled tree structure, store is a sibling of prefsEditorLoader. So the configuration for store comes from their parent component that is the top level of the builder constructed component.

Customizing store options this way is the API we provide to integrators, as well as customizing other parts like enhancer. I think it's OK to have this info in the demo index file. If you do think we should hide this information in index.html, we can create a component that holds these information and run builder so that the only thing index.html does is to instantiate that component.